brink/the-bitcoin-development-podcast/discussing-0-21-0-bitcoin-core-vulnerability-disclosures.md

```diff
@@ -63,54 +64,52 @@ One is that the active nodes on the network and their IP addresses or onion addr
 So if we tried to release software where we went and scraped all of the addresses of the active nodes on the network and tried to publish that along with the Bitcoin Core software, that would be outdated really, really quickly.
 And the second thing that makes this really difficult is, of course, like, because these peers are anonymous, you might imagine that there are malicious peers and people that are going to tell you about addresses, they may give you the addresses of just nodes that they control.
 And that can be really, really dangerous because if you join the network and let's say you connect to 20 other nodes and all of them are controlled by this one person who is say trying to attack you, you've been eclipsed.
-And something that they can do is they can either withhold the current main chain from you so you're not aware of new transactions and new blocks that come out, or they can even feed you an alternative chain where, For example, you got paid when you didn't actually get paid in the real Bitcoin blockchain.
+And something that they can do is they can either withhold the current main chain from you so you're not aware of new transactions and new blocks that come out, or they can even feed you an alternative chain where, for example, you got paid when you didn't actually get paid in the real Bitcoin blockchain.
 And so it's very important that your contact book, your address book, has hopefully honest, good peers in there.
 And so, on the node, sorry, on the network, nodes will gossip adders to each other, addresses.
-So this is different from your Bitcoin address, like BC1, blah blah blah.
+So this is different from your Bitcoin address, like BC1.
 This is the addresses, network addresses, like IP addresses or onion addresses.
 So just make sure that you don't confuse those.
 We're going to call them adders.
```

Comment:
This refers to `addr` messages. And "adders" should be replaced with `addrs` across the transcript.